src/js/config/OperationConfig.js

```diff
@@ -2993,5 +2993,26 @@ var OperationConfig = {
                 value: MorseCode.WORD_DELIM_OPTIONS
             }
         ]
+    },
+    "Tar": {
+        description: "Packs the input into a tarball.<br><br>No support for multiple files at this time.",
+        run: Compress.tar,
```

Comment:
Rename function to `runTar` as per conventions for other operations. I should probably document this more clearly somewhere. Its not massively important but it makes it clearer which functions are the actual operations when reading through the `Compress` namespace.